experimental/traffic-portal/src/app/shared/tree-select/tree-select.component.html

```diff
@@ -0,0 +1,47 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<div class="tree-select-root" role="combobox">
+	<mat-form-field>
+		<mat-label>{{label}}</mat-label>
+		<input type="text" id="{{handle}}-tree-select" name="{{handle}}-tree-select" matInput required readonly (click)="toggle($event)" [(ngModel)]="selected.name" [disabled]="disabled"/>
```

Comment:
Because this input is disabled, it has `cursor: not-allowed`, because of the global stylesheet rule. But actually, clicking on it is a valid action that does something useful, so it should probably have e.g. `cursor: pointer`